File: Selenium/Activities/Activity7.py
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as chromeService
from webdriver_manager.chrome import ChromeDriverManager
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys
from selenium.webdriver import ActionChains
service=chromeService(ChromeDriverManager().install())
with webdriver.Chrome(service=service) as driver:
    actions = webdriver.ActionChains(driver)
    driver.get("https://www.v1.training-support.net/selenium/drag-drop")
    pageTitle=driver.title
    print(pageTitle)
    football =driver.find_element(By.XPATH,"//img[@id='draggable']");
    dropzone1 =driver.find_element(By.ID,"droppable");
    dropzone2 =driver.find_element(By.ID,"dropzone2");
    actions.drag_and_drop(football,dropzone1).perform()
    actions.drag_and_drop(football,dropzone2).perform()
    driver.close()

File: Selenium/Activities/Activity7.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;
public class Activity7 {
    public static void main(String[] args) {
        WebDriverManager.chromedriver().setup();
        WebDriver driver = new ChromeDriver();
        Actions builder = new Actions(driver);
        driver.get("https://v1.training-support.net/selenium/drag-drop");
        System.out.println("Title of webpage : " + driver.getTitle());
        WebElement football =driver.findElement(By.xpath("//img[@id='draggable']"));
        WebElement dropzone1 =driver.findElement(By.id("droppable"));
        WebElement dropzone2 =driver.findElement(By.id("dropzone2"));
        builder.clickAndHold(football).moveToElement(dropzone1).pause(1000).release().build().perform();
        builder.dragAndDrop(football,dropzone2).build().perform();
        driver.close();


    }
}